sys-cluster/rdma-core/rdma-core-34.1.ebuild

```diff
@@ -76,6 +76,7 @@ src_configure() {
 		-DENABLE_VALGRIND="$(usex valgrind)"
 		-DENABLE_RESOLVE_NEIGH="$(usex neigh)"
 		-DENABLE_STATIC="$(usex static-libs)"
+		-DNO_PYVERBS="$(usex python OFF ON)"
 	)
 
 	if use python; then
```

Comment:
If you do that, you should remove this if statement, no?

Reply:
Oops, sorry, forgot to remove those other lines, apparently. I'll fix that...